shopinvader/models/shopinvader_variant.py

```diff
@@ -176,60 +176,97 @@ def _compute_variant_attributes(self):
                 variant_attributes[sanitized_key] = att_value.name
             record.variant_attributes = variant_attributes
 
-    def _get_price(self, pricelist, fposition, company=None):
-        self.ensure_one()
-        return self._get_price_per_qty(1, pricelist, fposition, company)
+    def _get_price(self, qty=1.0, pricelist=None, fposition=None, company=None):
+        """Computes the product prices
 
-    def _get_price_per_qty(self, qty, pricelist, fposition, company=None):
```

Comment:
Not sure that we should remove this method.

Indeed there is a module (V8 only) that add a extra key for getting the price per qty see here : https://github.com/shopinvader/odoo-shopinvader/blob/8.0/shopinvader_price_per_qty/models/product.py

So the idea was to overwrite the _get_price method and then call X time the method _get_price_per_qty

Reply:
There's no need for 2 different methods for that module to work, if it gets migrated to `14.0`.

Something like this could be done:

```py
def _get_price(self, qty=1.0, pricelist=None, fposition=None, company=None):
    res = super()._get_price(
        qty=qty, pricelist=pricelist, fposition=fposition, company=company
    )
    res["price_per_qty"] = {}
    if qty == 1.0:
        items = get_pricelist_items()  # pseudo-code
        items_qty = set(item.min_quantity for item in items if item.min_quantity > 1)
        for item_qty in items_qty:
            res["price_per_qty"][qty] = self._get_price(
                qty=item_qty, pricelist=pricelist, fposition=fposition, company=company
            )
```

Reply:
Indeed it should be fine